packages/cloud_functions/cloud_functions_web/README.md

```diff
@@ -0,0 +1,82 @@
+# cloud_functions_web
+
+The web implementation of [`cloud_functions`][1].
+
+## Usage
+
+### Import the package
+
+This package is the endorsed implementation of `cloud_functions` for the web platform since version `0.0.1`, so it gets automatically added to your dependencies by depending on `cloud_functions: ^0.0.1`.
```

Comment:
Put a TODO here to update this with the correct version number as soon as this package actually becomes an endorsed implementation of `package:cloud_functions`

Reply:
Oh, good catch. Sorry, will do that right now.